test/map-symbolic/remove-9-spec.k

```diff
@@ -3,12 +3,8 @@
 
 requires "map-tests.k"
 
-module VERIFICATION
-    imports MAP-TESTS
-endmodule
-
 module REMOVE-9-SPEC
-    imports VERIFICATION
+    imports MAP-TESTS
```

Comment:
Did you modify https://github.com/kframework/kore/blob/3eefcb078b24128e5773af9dd1170a19230dd832/test/map-symbolic/gen-tests.sh?

For these tests in map-symbolic, you must modify that generator script, and generate them with the script.

Reply:
How should that script be ran? `./gen-tests.sh` doesn't have effect

Reply:
How should that script be ran? ./gen-tests.sh doesn't have effect